src/Core/Form/ChoiceProvider/TranslationTypeChoiceProvider.php

```diff
@@ -60,4 +60,14 @@ public function getChoices()
             $this->translator->trans('Other translations', [], 'Admin.International.Feature') => 'others',
         ];
     }
+
+    public function getExportCoreChoices(): array
+    {
+        return [
+            $this->translator->trans('Back office translations', [], 'Admin.International.Feature') => 'back',
+            $this->translator->trans('Front office Translations', [], 'Admin.International.Feature') => 'front',
```

Comment:
```suggestion
            $this->translator->trans('Front office translations', [], 'Admin.International.Feature') => 'front',
```

Reply:
Unfortunately, we have a capital T on crowdin. That's why I keep it

Reply:
I think you must change. Crowndin will be changed.